src/__tests__/pet.test.js

```diff
@@ -0,0 +1,168 @@
+const { Pet } = require ('../pet.js');
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :(';
+
+let fido;
+beforeEach(() => {
+    fido = new Pet('Fido');
+});
+
+describe ('Pet', () => {
+
+  test('Returns an instance of Pet', () => {
+    expect(fido).toBeInstanceOf(Pet);
+  });
+
+  test('Returns the pets name', () => {
+    expect(fido.name).toBe('Fido');
+  });
+});
+
+describe ('growUp', () => {
+  
+  test ('has an initial age of 0', () => {
+    expect (fido.age).toEqual (0);
+   });
+
+   test ('increments the age by 1', () => {
+    fido.growUp();
+    expect (fido.age).toEqual (1);
+  });
+  
+  test ('has an initial hunger of 0', () => {
+    expect (fido.hunger).toEqual (0);
+  })
+
+  test ('increments the hunger by 5', () => {
+    fido.growUp();
+    expect (fido.hunger).toEqual (5);
+  });
+
+  test ('has an initial fitness of 10', () => {
+    expect (fido.fitness).toEqual (10);
```

Comment:
You could also move this test into the previous describe block as the initial fitness is not set by the growUp method.